pandas/core/window.pyx

```diff
@@ -307,18 +322,35 @@ def get_window_indexer(input, win, minp, index, floor=None,
         compat Indexer that allows us to use a standard
         code path with all of the indexers.
 
+
     Returns
     -------
     tuple of 1d int64 ndarrays of the offsets & data about the window
 
     """
 
+    cdef:
+        bint l_closed = False
+        bint r_closed = False
+
+    if closed not in ['right', 'left', 'both', 'neither']:
```

Comment:
you can just do an assert here (this is an internal routine that has certain guarantees)